cursorless-talon-dev/src/cursorless_dev_vscode.talon

```diff
@@ -11,3 +11,8 @@ app: vscode
     user.vscode("workbench.action.tasks.terminate")
     insert("Serve")
     key(enter)
+
+^debug generate subset$:
+    user.vscode_with_plugin("workbench.action.tasks.runTask", "Generate test subset file")
+^debug edit subset$:
+    user.vscode_with_plugin("commands.openFolder", "./packages/common/src/testUtil/testSubsetGrep.properties")
```

Comment:
I split this into two commands because the version that runs as a task was painfully slow on my computer. Which is fine for the initial generation, but too slow for something I do many times a day. This version is ~instant

Would be great if we could get away with one command that was fast, but I couldn't figure out how to do it 🤷‍♂️

Reply:
ack